src/main/java/org/dcsa/tnt/model/base/AbstractEventSubscription.java

```diff
@@ -7,25 +7,70 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.relational.core.mapping.Column;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 import java.util.UUID;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class AbstractEventSubscription extends AuditBase {
 
+    public String getBookingReference() {
+        return carrierBookingReference;
+    }
+
+    public void setBookingReference(String bookingReference) {
+        this.carrierBookingReference = bookingReference;
+    }
+
     @Id
     @Column("subscription_id")
     private UUID subscriptionID;
 
     @Column("callback_url")
+    @NotNull
+    @NotEmpty
     private String callbackUrl;
 
+
     @Column("booking_reference")
     private String bookingReference;

```

Comment:
Don't we need to remove `booking_reference` , since `carrier_booking_reference` is added ? As I see its been removed from `event_subscription` in [05_reference_implementation_support](https://github.com/dcsaorg/DCSA-Information-Model/pull/104/files)

Reply:
That is an error, it should be included. I will fix that now.

Reply:
Sure, could you also rebase [DCSA-Information-Model](https://github.com/dcsaorg/DCSA-Information-Model/compare/8feacb97949dfca1f57d09b65eb95a7313992694...b31559089ed18137d47c08b64700a20945bc959a#diff-d47a2bfa5801ca0d14c0ebdab44ad9d93f41e78b831c791bee09328e3755aa86) with it's current master so its up to date and history is consistent?

Reply:
Yes, will do.